File: _src/06/Chapter6.CSharp/Models/Company.cs

using System;
using System.Collections.Generic;

namespace Chapter6.CSharp.Models
{
    public class Company
    {
        public Company()
        {
            Persons = new HashSet<Person>();
        }
        public int CompanyId { get; set; }
        public string CompanyName { get; set; }
        public DateTime DateAdded { get; set; }
        public bool IsActive { get; set; }
        public ICollection<Person> Persons { get; set; }
    }
}

File: _src/06/Chapter6.VB/Models/Company.vb
Namespace Models

    Public Class Company
        Public Sub New
            Persons = new HashSet(Of Person)
        End Sub
        
        Property CompanyId() As Integer
        Property CompanyName() As String
        Property DateAdded() As DateTime
        Property IsActive() As Boolean
        Overridable Property Persons() As ICollection(Of Person)
    End Class
End Namespace